### PR TITLE
combined.cc implementing customNetDevice

### PR DESCRIPTION
**Pull request by charliebarber**
Getting this error:
```
FRRQueue:GetPointToPointDevice()
FRRQueue:GetDevice()
FRRQueue:GetPointToPointDevice()
FRRQueue:GetQueue(0xaaaac81c52d0)
NS_ASSERT failed, cond="m_ptr", msg="Attempted to dereference zero pointer", +0.000000000s -1 file=/usr/workspace/ns-allinone-3.41/ns-3.41/src/core/model/ptr.h, line=713
NS_FATAL, terminating
terminate called without an active exception
Command 'build/scratch/ns3.41-combined-default' died with <Signals.SIGABRT: 6>.
```

### CODE REVIEW

File: src/combined.cc [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+    toggleCongestion(getQueue<0>(devices_0_2));
+    toggleCongestion(getQueue<1>(devices_0_2));
+    toggleCongestion(getQueue<0>(devices_2_4));
+    toggleCongestion(getQueue<1>(devices_2_4));
+    toggleCongestion(getQueue<0>(devices_4_3));
+    toggleCongestion(getQueue<1>(devices_4_3));
+    toggleCongestion(getQueue<0>(devices_3_5));
+    toggleCongestion(getQueue<1>(devices_3_5));
+    toggleCongestion(getQueue<0>(devices_1_2));
+    toggleCongestion(getQueue<1>(devices_1_2));
```
Comment (YousefEZ) [inline src/combined.cc]: some of these devices are not using a simulation queue, and could trigger a segfault.

e.g. `devices_0_2`

Comment (YousefEZ) [reply src/combined.cc]: only `devices_2_3` is has a simulation queue

**Comment by charliebarber** [comment]
Ready for review. Combined.cc now ready for experimental setup.

**Review by YousefEZ — APPROVED** [review]
LGTM well done!

**Pull request merged**